.eslintrc.cjs

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  rules: {
+    'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    'no-console': 'off',
```

Comment:
we can  enable it if consider using winston or any other logger tool

Reply:
I believe we can do it. We’ll talk with a team about it today.